server.go

```diff
@@ -660,6 +660,7 @@ func newServer(listenAddrs []net.Addr, chanDB *channeldb.DB,
 			AprioriHopProbability: routingConfig.AprioriHopProbability,
 			PenaltyHalfLife:       routingConfig.PenaltyHalfLife,
 			MaxMcHistory:          routingConfig.MaxMcHistory,
+			AprioriWeight:         routing.DefaultAprioriWeight,
```

Comment:
We should add this to the routing config so we can experiment with different a prior weight values without having to recompile the target `lnd` binary.

Reply:
Definitely needs to be configurable. This is _the_ parameter that is going to shrink the graph, hopefully :) Fixed